code/modules/admin/verbs/playsound.dm

```diff
@@ -174,6 +181,93 @@
 
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Play Internet Sound")
 
+/proc/var_internet_sound(mob/usr, request)
```

Comment:
rename this to something not shit.

Reply:
also this entire proc is not allowed levels of copypasta.

you should split up play internetsound between the proc that plays the sound and the proc that requests the sound from the admin in a prompt, then this proc becomes unneeded.